pygmt/tests/test_timestamp.py

```diff
@@ -86,3 +86,38 @@ def test_timestamp_text(faketime):
     fig = Figure()
     fig.timestamp(text=faketime)
     return fig
+
+
+@pytest.mark.mpl_image_compare(filename="test_timestamp.png")
+def test_timestamp_deprecated_timestamp(faketime):
+    """
+    Check if the deprecated parameter 'timestamp' works but raises a warning.
+    """
+    fig = Figure()
+    with pytest.warns(expected_warning=SyntaxWarning) as record:
+        with config(FORMAT_TIME_STAMP=faketime):
+            # plot nothing (the data is outside the region) but a timestamp
+            fig.plot(
+                x=0,
+                y=0,
+                style="p",
+                projection="X1c",
+                region=[1, 2, 1, 2],
+                timestamp=True,
+            )
+        assert len(record) == 1  # check that only one warning was raised
```

Comment:
```suggestion
        assert len(record) == 1 # check that only one warning was raised
```

Reply:
PEP8 required at least two whitespaces between statement and comment (https://peps.python.org/pep-0008/#inline-comments).

> An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

Reply:
Ok 👌